doc/server_configurations.md

```diff
@@ -7695,14 +7695,50 @@ Volar can be installed via npm
 npm install -g @volar/server
 ```
 
-With Vue 3 projects - it works out of the box.
+Vue 3 works by default, but Vue 2 needs [additional configuration](https://github.com/johnsoncodehk/volar/blob/master/extensions/vscode-vue-language-features/README.md?plain=1#L28-L63)
 
-With Vue 2 projects - requires [additional configuration](https://github.com/johnsoncodehk/volar#using)
+`vuels` and `volar` shouldn't be running at the same time
```

Comment:
avoid contractions in the documentation

Reply:
Also maybe add explanation as to why they should not be running at the same time (don't call setup{} on both) etc.

Reply:
I've rewritten it with explanation and removed contraction